README.md

```diff
@@ -1,2 +1,157 @@
 # cesium-webpack-example
-The minimal recommended setup for an application using Cesium with Webpack.
+
+The minimal recommended setup for an application using [Cesium](https://cesiumjs.org/) with [Webpack](https://webpack.js.org/concepts/).
+
+[![Build Status](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example.svg?branch=using-custom-loader)](https://travis-ci.org/AnalyticalGraphicsInc/cesium-webpack-example)
+
+### Running this application
+
+	npm install
+	npm start
+
+##### Available Scripts
+
+* `npm start` - Runs a webpack build with `webpack.config.js` and starts a development server
+* `npm run build` - Runs a webpack build with `webpack.config.js`
+* `npm run release` - Runs an optimized webpack build with `release.webpack.congif.js`
+
+### Requiring Cesium in your application
+ 
+##### CommonJS Require
+
+ 	var Cesium = require('cesium/Cesium');
+ 	var viewer = new Cesium.Viewer('cesiumContainer');
+
+##### ES6 Style Import
+
+ 	import Cesium from 'cesium/Cesium';
```

Comment:
Is there a way to require a specific module ES6 style?

Reply:
Yes, I'll add it